docs/docs/how_to/pebblo_retrieval_qa.ipynb

```diff
@@ -0,0 +1,488 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "3ce451e9-f8f1-4f27-8c6b-4a93a406d504",
+   "metadata": {},
+   "source": [
+    "# Identity-enabled RAG using PebbloRetrievalQA\n",
+    "\n",
+    "> PebbloRetrievalQA is a Retrieval chain with Identity & Semantic Enforcement for question-answering\n",
+    "against a vector database.\n",
+    "\n",
+    "This notebook covers how to retrieve documents using Identity & Semantic Enforcement (Deny Topics/Entities).\n",
```

Comment:
Add a sentence/link here such as "You can also visit <link> for more details on Pebblo and its SafeRetriever feature"

link: https://daxa-ai.github.io/pebblo/retrieval_chain